icc-x-api/icc-calendar-item-x-api.ts

```diff
@@ -20,19 +20,22 @@ export class IccCalendarItemXApi extends iccCalendarItemApi {
   }
 
   newInstance(user: UserDto, ci: CalendarItemDto) {
-    const calendarItem = _.extend(
-      {
-        id: this.crypto.randomUuid(),
-        _type: "org.taktik.icure.entities.CalendarItem",
-        created: new Date().getTime(),
-        modified: new Date().getTime(),
-        responsible: user.healthcarePartyId,
-        author: user.id,
-        codes: [],
-        tags: []
-      },
-      ci || {}
+    const calendarItem = new models.CalendarItemDto(
```

Comment:
`new models.*Dto` should never be used. We loose the typing due to the Swagger generated code accepting any in the constructor.